test/extended/util/framework.go

```diff
@@ -793,7 +793,7 @@ func WaitForAnImageStreamTag(oc *CLI, namespace, name, tag string) error {
 // TimedWaitForAnImageStreamTag waits until an image stream with given name has non-empty history for given tag.
 // Gives up waiting after the specified waitTimeout
 func TimedWaitForAnImageStreamTag(oc *CLI, namespace, name, tag string, waitTimeout time.Duration) error {
-	g.By(fmt.Sprintf("waiting for an is importer to import a tag %s into a stream %s", tag, name))
+	g.By(fmt.Sprintf("waiting for tag %s in image stream %s/%s", tag, namespace, name))
```

Comment:
`waiting for tag %s in imagestream %s  to be imported`

or you could just change `is` to `imagestream` in the original text.

Reply:
> `waiting for tag %s in imagestream %s/%s to be imported`

Done, and rebased onto master with 0b8b43c8f3 -> c29c1f6364.